### ISSUE TITLE
Is it possible to reference an image file from the same directory as the MarkDown file?

### ISSUE THREAD

**Issue by notalentgeek** [gilbitron/Raneto#161]
This is just for convenience sake, mostly. I have a lot of notes with a lot of images. Set images from `%image_url%` is not as convenience than having the image in the same directory as the MarkDown (.md) files.

So instead of this.

`![%image_url%/image-1.png](%image_url%/image-1.png)`

I will just need to do this.

`![./image-1.png](./image-1.png)`

Is this possible?

**Comment by furier** [comment]
I would like to see this functionality as well.

**Comment by tobyqin** [comment]
1+ request.

**Comment by philipstratford** [comment]
I agree that having to put all of the images relating to your documentation (ours will have hundreds) in the images folder very separate from the content is not ideal. Also, the fact that the designated images folder is located inside `themes/default/public` very much suggests that it's intended for images relating to the site's overall theme, not the actual content itself.

A location for images inside the `content` folder would make more sense (the images are as much part of the documentation's content as the text), and the ability to have images stored basically anywhere within the `content` folder and referenced in documents using relative paths would allow me to organise my images with my `.md` files, which would be much more convenient.

**Comment by cardinalfan1** [comment]
Agree with this post. Inline images would be very helpful

**Comment by mladenmilev** [comment]
Images are very much needed as essencial part of any documentation

**Comment by cyberience** [comment]
I would also say that, even as a work around, having image subfolders in the images folder would partly help, so that one can at least tage the images into the respective locations.

**Comment by cyberience** [comment]
Its probably useful to note that, many IDE where you can edit markdown, can display the image inline when its in the relative path.. but having to place images into an image folder means the IDE for example in webstorm will not represent the edited page since it is not correctly referencing a rooted image path. This is open source project, and I am not a javascript programmer, so hopefully someone would like to take on this challenge. since this issue has been pending 5 years already. maybe quicker if I lean javascript. lol
